engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -573,8 +581,30 @@ - (BOOL)containsWebView:(UIView*)view remainingSubviewDepth:(int)remainingSubvie
   return NO;
 }
 
+- (UIView*)hitTest:(CGPoint)point withEvent:(UIEvent*)event {
+  if (_blockingPolicy == FlutterPlatformViewGestureRecognizersBlockingPolicyTouchBlockingOnly) {
+    // In release mode, FlutterTouchInterceptingView's init is called before flutterViewController
+    // is set on platformViewsController.
+    if (self.flutterViewController == nil) {
+      _flutterViewController = self.platformViewsController.flutterViewController;
+    }
+    CGPoint pointInFlutterView = [self convertPoint:point toView:self.flutterViewController.view];
+    // Block gesture if the framework instructed so (after performing its own hitTest).
```

Comment:
This confused me at first, perhaps to make it more clear?

```suggestion
    // Consult the framework on if the touch should be handled by the platform view. 
    // If NO, the touch is handled by a Flutter widget and should be blocked (by returning self).
    // If YES, the touch should continue to the standard hit-testing (through super), allowing the touch to be delivered to the underlying native platform view or one of its subviews.
```

Reply:
done